### PR TITLE
Revision of Advanced Compositions

### LINKED ISSUE #146: Revision of Custom Compositions
- [ ] Feature: Tree control with search function.
- [ ] Change: Merge modules to a singe module.
- [ ] Feature: Support AI.
- [ ] Feature: Support

### PR DESCRIPTION
**Pull request by oOKexOo**
**When merged this pull request will:**
- Fix #146

### CODE REVIEW

File: @AresModAchillesExpansion/addons/functions_f_achilles/cfgFunctions.hpp [hunk 1]
```diff
@@ -111,6 +111,15 @@ class CfgFunctions
...
             class changeAccessoires;
 		};
+		class setAttribute
+		{
+			class setAttribute_group_behaviour {file = "\achilles\functions_f_achilles\functions\setAttribute\group\fn_behaviour.sqf"};
+
+			class setAttribute_unit_ability {file = "\achilles\functions_f_achilles\functions\setAttribute\unit\fn_ability.sqf"};
+			class setAttribute_unit_animation {file = "\achilles\functions_f_achilles\functions\setAttribute\unit\fn_animation.sqf"};
+			class setAttribute_unit_loadout {file = "\achilles\functions_f_achilles\functions\setAttribute\unit\fn_loadout.sqf"};
+
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/cfgFunctions.hpp]: Space.

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/group/fn_behaviour.sqf [hunk 2]
```diff
@@ -0,0 +1,20 @@
+params
+[
+	["_group", grpNull, [grpNull]],
+	["_params", [], [[]]]
+];
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/group/fn_behaviour.sqf]: Spacing

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/group/fn_behaviour.sqf [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+_params params
+[
+	["_combatMode", "", [""]],
+	["_behaviour", "", [""]]
+];
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/group/fn_behaviour.sqf]: Spacing

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/unit/fn_ability.sqf [hunk 4]
```diff
@@ -0,0 +1,34 @@
+params
+[
+	["_unit", objNull, [objNull]],
+	["_params", [], [[]]]
+];
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/unit/fn_ability.sqf]: Spacing

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/unit/fn_ability.sqf [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+{
+	_x params
+	[
+		"_ability", 
+		["_enabled", true, [false]]
+	];
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/unit/fn_ability.sqf]: Spacing

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/unit/fn_animation.sqf [hunk 6]
```diff
@@ -0,0 +1,13 @@
+params
+[
+	["_unit", objNull, [objNull]],
+	["_params", [], [[]]]
+];
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/unit/fn_animation.sqf]: Spacing

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/unit/fn_loadout.sqf [hunk 7]
```diff
@@ -0,0 +1,13 @@
+params
+[
+	["_unit", objNull, [objNull]],
+	["_loadout", [], [[]]]
+];
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/unit/fn_loadout.sqf]: Spacing

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/group/fn_behaviour.sqf (hunk 2 above)
Comment (oOKexOo) [reply @AresModAchillesExpansion/addons/functions_f_achilles/functions/setAttribute/group/fn_behaviour.sqf]: Spacing? Are you kidding me? This is how we have it almost everywhere.

File: @AresModAchillesExpansion/addons/functions_f_achilles/cfgFunctions.hpp (hunk 1 above)
Comment (oOKexOo) [reply @AresModAchillesExpansion/addons/functions_f_achilles/cfgFunctions.hpp]: Personally, I have a better overview of what functions we have when it's a one-liner here. It's going to be a rather long list.

Comment (CreepPork) [reply @AresModAchillesExpansion/addons/functions_f_achilles/cfgFunctions.hpp]: That's not the issue. Look one line down and it's just an empty line between the `}` bracket. It should be removed.

**Pull request closed without merging**